### DISCUSSION TITLE
欢迎来讨论

### DISCUSSION THREAD

**Discussion by ZainCheung** [ZainCheung/netease-cloud#77] [General]
为了更好的进行讨论，说明下本讨论区规则

-只讨论项目相关的问题

-不要闲聊

-不要发布广告，维护良好的社区环境，谢谢。

**Comment by 2571003556** [comment]
云函数报错-- START RequestId:bf89ae2d-fd1e-4890-9c79-fd6e8c3fa460

ERROR RequestId:bf89ae2d-fd1e-4890-9c79-fd6e8c3fa460 Result:{"errorCode":1,"errorMessage":"user code exception caught","stackTrace":"Traceback (most recent call last):\n  File \"/var/user/index.py\", line 331, in main\n    taskPool()\n  File \"/var/user/index.py\", line 308, in taskPool\n    config = init()\n  File \"/var/user/index.py\", line 256, in init\n    appToken = config['setting']['appToken']\n  File \"/var/lang/python3/lib/python3.6/configparser.py\", line 1233, in __getitem__\n    raise KeyError(key)\nKeyError: 'appToken'","statusCode":430}
这个是啥问题啊，更换2.7的环境也不行
